asyncpg/connection.py

```diff
@@ -1231,6 +1236,47 @@ def _drop_global_statement_cache(self):
         else:
             self._drop_local_statement_cache()
 
+    def drop_cache(self):
+        """Drop internally cached objects in the connection.
```

Comment:
```python
"""Indicate that the database schema information must be reloaded.

For performance reasons, asyncpg caches certain aspects of the 
database schema, such as the layout of composite types.  Consequently,
when the database schema changes, and asyncpg is not able to
gracefully recover from an error caused by outdated schema 
assumptions, an :exc:`~asyncpg.exceptions.OutdatedSchemaCacheError` 
is raised.  To prevent the exception this method may be used to inform 
asyncpg that the database schema has changed.

Example:
...
"""
```